server/apps/immich/src/api-v1/album/album.controller.ts

```diff
@@ -33,25 +33,29 @@ import {
   IMMICH_CONTENT_LENGTH_HINT,
 } from '../../constants/download.constant';
 import { DownloadDto } from '../asset/dto/download-library.dto';
+import { CreateAlbumShareLinkDto as CreateAlbumSharedLinkDto } from './dto/create-album-shared-link.dto';
 
 // TODO might be worth creating a AlbumParamsDto that validates `albumId` instead of using the pipe.
-@Authenticated()
```

Comment:
There is another way to use decorators that would allow that, but not how we currently have it. Probably in another PR we can refactor the auth guard, and then it would allow for that.